### ISSUE TITLE
CMakeLists - for your review

### ISSUE THREAD

**Issue by tdiprima** [InsightSoftwareConsortium/ITK-OpenCV-Bridge-Tutorial#1]
Hey there!  Thanks for the tutorial.  There are 3 CMakeLists files (locations below) that are using the old deprecated USE_ITK_FILE.  I think you want us to use ITK_USE_FILE instead?
- ITK-OpenCV-Bridge-Tutorial/Exercises/ITKIntroduction/exercise1/CMakeLists.txt
- ITK-OpenCV-Bridge-Tutorial/Exercises/ITKVideoPipeline/CMakeLists.txt
- ITK-OpenCV-Bridge-Tutorial/Exercises/ITKOpenCVBridge/CMakeLists.txt

If we don't change the variable to ITK_USE_FILE, CMake throws an error:

```
CMake Error at Exercises/ITKIntroduction/exercise1/CMakeLists.txt:10 (include):
  include called with wrong number of arguments.  include() only takes one
  file.
```

Thanks.

**Comment by thewtex** [comment]
@tdiprima Thanks for reporting this issue. Yes, it should be `ITK_USE_FILE`. Please do not hesitate to make a pull request.

**Comment by tdiprima** [comment]
Thanks!  I checked out the code to a new branch.  Committed the changes, locally.  And then I think I'm supposed to do a `git push --set-upstream origin fixCMakeLists` (the branch name), but when I did that it said `Permission denied`.
Clicking the green "new pull request" button on the repo, wasn't happy with me trying to make a pull request either.

**Comment by thewtex** [comment]
Try

```
git remote add tdiprima git@github.com:tdiprima/ITK-OpenCV-BridgeTutorial
git fetch tdiprima
git push tdiprima fixCMakeLists
```

Then create a pull request through the GitHub web interface.

Some more general GitHub workflow guidelines:

https://guides.github.com/

**Comment by thewtex** [comment]
Addressed by #2
